.github/workflows/ms.analysisservices.servers.yml

```diff
@@ -112,7 +117,7 @@ jobs:
     if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/master' || github.event.inputs.prerelease == 'true'
     runs-on: ubuntu-20.04
     needs:
-      - job_set_workflow_param
+      - job_prepare_pipeline
```

Comment:
Looks like we don't need this anymore for the publishing step

Reply:
Removed